models/fates/R/write.configs.FATES.R

```diff
@@ -52,56 +45,60 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
 
    ## SITE INFO --> DOMAIN FILE (lat/lon)
    #  - should also store this in the refcase directory for PEcAn so we can grab from there, and not the PEcAn package
-   gridres <- 0.125  ## ultimately this should be a variable
-   lat <- as.numeric(site$lat)
-   lon <- (as.numeric(site$lon) + 360) %% 360 ## make sure coords in 0-360 range, not negative
-   domain.default <- system.file("domain.lnd.1x1pt-brazil_navy.090715.nc",package="PEcAn.FATES")
-   domain.file <- file.path(local.rundir,paste0("domain.lnd.",site_name,".nc"))
-   file.copy(domain.default,domain.file)
-   domain.nc <- ncdf4::nc_open(domain.file,write=TRUE)
-   ncdf4::ncvar_put(nc=domain.nc, varid='xc', vals=lon)
-   ncdf4::ncvar_put(nc=domain.nc, varid='yc', vals=lat)
-   ncdf4::ncvar_put(nc=domain.nc, varid='xv', vals=lon+c(-1,1,1,-1)*gridres)
-   ncdf4::ncvar_put(nc=domain.nc, varid='yv', vals=lat+c(-1,-1,1,1)*gridres)
-   ncdf4::ncvar_put(nc=domain.nc, varid='area', vals=(2*gridres*pi/180)^2)   
-   ncdf4::nc_close(domain.nc)
+   #gridres <- 0.125  ## ultimately this should be a variable
```

Comment:
Is the DOMAIN file no longer needed? Not following why this  bit is being commented out?

Reply:
@mdietze Yes, for single site simulations, domain files become optional in the recent versions of CLM with new coupler [nuopc](https://earthsystemmodeling.org/nuopc/).